packages/map/package.json

```diff
@@ -21,7 +21,7 @@
     "prepublishOnly": "rollup-type-bundler"
   },
   "dependencies": {
-    "@joshdb/core": "next",
+    "@joshdb/core": "3.0.0-next.4613706cda66569b2d1dc99f06665eecec2505ce.0",
```

Comment:
Why is this 3.0.0 and the rest 1.1.0?

Reply:
That's just what the version resolved to when being bumped. Nothing to be worried about.